packages/waku/tests/vite-plugin-rsc-analyze.test.ts

```diff
@@ -2,16 +2,12 @@ import { build } from 'vite';
 import { expect, test, describe } from 'vitest';
 import { fileURLToPath } from 'node:url';
 import path from 'node:path';
-import crypto from 'node:crypto';
 import type { LoggingFunction, RollupLog } from 'rollup';
 
 import { rscAnalyzePlugin } from '../src/lib/plugins/vite-plugin-rsc-analyze.js';
 
 const root = fileURLToPath(new URL('./fixtures', import.meta.url));
 
-// FIXME vitest node environment does not have crypto
-globalThis.crypto = crypto as any;
```

Comment:
This didn't seem needed anymore. Curious if using node 20 is what fixes this?

I found this related discussion: https://github.com/vitest-dev/vitest/discussions/893

but fwiw this is the output when I `console.log(globalThis.crypto)` when running `vitest --silent=false`

<img width="383" alt="testsvite-plugin-rsc-analyze test ts129" src="https://github.com/user-attachments/assets/0584bdf2-9846-4f9d-a821-859ad533fdcc">

happy to leave this out of the PR, but seems good to remove if it is truly no longer an issue

Reply:
So this fails in CI using node 18.17.0!

I was able to replicate this locally as well... so that begs the question, do we need to support node 18, or would upgrading the version used in CI and in the engines field of the package.json to all use node 20 be acceptable?

Or would you prefer we just update the comment here mentioning that this issue is specific to node 18 and no longer an issue in node 20?

Reply:
Yes, this is for node 18. Feel free to update the comment.
We may drop node 18 in the future and run CI with node 20 and node 22.